bin/hjs-dev-server.js

```diff
@@ -21,7 +22,21 @@ try {
 }
 
 var serverConfig = config.devServer
+var https = serverConfig.https
 var app = express()
+
+var createServer = require(https ? 'https' : 'http').createServer
+var server
+
+if (https) {
+  server = createServer({
+    key: fs.readFileSync(path.resolve(__dirname, '../resources/hjs-webpack-localhost.key')),
+    cert: fs.readFileSync(path.resolve(__dirname, '../resources/hjs-webpack-localhost.crt'))
+  }, app)
```

Comment:
I think it would be good to add the ability to make `https` an options object that is passed to `createServer`, sth like this:

```js
if (https) {
  var config = {
    key: fs.readFileSync(..),
    cert: fs.readFileSync(..)
  }

  if (typeof https === 'object') {
    config = Object.assign(config, https)
  }
  server = createServer(config, app)
}
```